### ISSUE TITLE
Doorbell - no names on accessories

### ISSUE THREAD

**Issue by jfmach** [hjdhjd/homebridge-unifi-protect#927]
**Describe The Problem:**
I have the exact same problem than the one reported in https://github.com/hjdhjd/homebridge-unifi-protect/issues/921
Doorbell accessories have no name so you don't know which one does what.

**To Reproduce:**
Systematic, Doorbell accessories have no names, whether in Home App or in Eve.

**Logs:**
```[22/04/2023, 16:21:14] Loaded plugin: homebridge-unifi-protect@6.4.4
[22/04/2023, 16:21:14] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[22/04/2023, 16:21:25] [homebridge-unifi-protect] Launched child bridge with PID 28147
[22/04/2023, 16:21:29] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[22/04/2023, 16:21:29] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.4.4 child bridge successfully
[22/04/2023, 16:21:34] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port edited.
[22/04/2023, 16:21:36] [homebridge-unifi-protect] edited: Connected to the UniFi Protect API at edited.
[22/04/2023, 16:21:36] [homebridge-unifi-protect] Video Recorder [UNVR4]: Enabled system information sensor: cpu temperature.
[22/04/2023, 16:21:36] [homebridge-unifi-protect] Video Recorder: Discovered nvr: Video Recorder [UNVR] (address: edited mac: edited).
[22/04/2023, 16:21:36] [homebridge-unifi-protect] Video Recorder: Discovered camera: 04. Front Door [G4 Doorbell] (address: edited mac: edited).
[22/04/2023, 16:21:36] [homebridge-unifi-protect] Video Recorder [UNVR4] 04. Front Door [G4 Doorbell]: HomeKit Secure Video event recording configured to use only: Low.
[22/04/2023, 16:21:37] [homebridge-unifi-protect] Video Recorder [UNVR4] 04. Front Door [G4 Doorbell]: Enabling doorbell automation trigger.
[22/04/2023, 16:21:37] [homebridge-unifi-protect] Video Recorder [UNVR4] 04. Front Door [G4 Doorbell]: Discovered doorbell message switch: LEAVE PACKAGE AT DOOR.
[22/04/2023, 16:21:37] [homebridge-unifi-protect] Video Recorder [UNVR4] 04. Front Door [G4 Doorbell]: Discovered doorbell message switch: DO NOT DISTURB.
[22/04/2023, 16:21:37] [homebridge-unifi-protect] Video Recorder [UNVR4] 04. Front Door [G4 Doorbell]: Doorbell message set: LEAVE PACKAGE AT DOOR.
[22/04/2023, 16:21:37] [homebridge-unifi-protect] Video Recorder: Default doorbell message set to: WELCOME.
[22/04/2023, 16:21:37] [homebridge-unifi-protect] Video Recorder [UNVR4] 04. Front Door [G4 Doorbell]: HomeKit Secure Video event recording enabled: 480x360@15fps (Low), 2000 kbps with a 4 second timeshift buffer.
```

**Homebridge Configuration:**

```json
        {
            "name": "Config",
            "port": edited,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "controllers": [
                {
                    "address": "edited",
                    "password": "edited",
                    "username": "edited"
                }
            ],
            "options": [
                "Enable.Nvr.SystemInfo",
                "Disable.Doorbell.Messages.FromDoorbell.edited",
                "Enable.Doorbell.Trigger.edited",
                "Enable.Video.HKSV.Record.Only.Low.edited"
            ],
            "_bridge": {
                "username": "edited",
                "port": edited
            },
            "platform": "UniFi Protect"
        }
```

**Screenshots:**
![IMG_5706](https://user-images.githubusercontent.com/5497534/233999456-e203e1e7-3a24-4389-bdea-f14152468b0a.PNG)
![IMG_5707](https://user-images.githubusercontent.com/5497534/233999461-6e4ffc4e-301e-45ae-a3b5-e4acf2d0fb6a.PNG)

**Environment:**

* **Homebridge Version:** 1.6.0
* **Node Version:** v18.14.2
* **Homebridge-UniFi-Protect Plugin Version:** 6.4.4
* **UniFi Protect Version:** 2.7.34
* **UniFi Protect Controller Type:** UNVR
* **UniFi Protect Controller Firmware Version:** I can't see a firmware version? If you mean the Unifi OS version it is UNVR v3.0.16
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iOS 16.4.1
* **Operating System and OS Version:** Debian 10.11

**Comment by hjdhjd** [comment]
This is a bug in iOS 16: https://github.com/homebridge/homebridge/issues/3210

Hopefully Apple gets to it at some point....

**Comment by jfmach** [comment]
Okay I see, thank you!
Workaround is to edit each accessory individually and delete the name, then you can see the actual default name:
<img width="504" alt="Screenshot 2023-04-24 at 1 41 02 PM" src="https://user-images.githubusercontent.com/5497534/234074644-b5bcb426-58f3-4728-b7fe-4b20db448879.png">
<img width="497" alt="Screenshot 2023-04-24 at 1 41 11 PM" src="https://user-images.githubusercontent.com/5497534/234074653-fe7b303e-6445-427e-b99e-d50b816d8759.png">

**Comment by hjdhjd** [comment]
Fixed in v6.5.0. Thanks much for the report.
